server/analytic.go

```diff
@@ -1,64 +0,0 @@
-package main
-
-import (
-	"sync"
-	"time"
-)
-
-type Analytic struct {
```

Comment:
If you have `NewAnalytic` this struct should be unexported.

Reply:
I'm not sure
![selection_194](https://user-images.githubusercontent.com/1492516/53493945-ea328780-3a9c-11e9-98d7-313fa28c5504.png)